prep-css-box-model/style.css

```diff
@@ -0,0 +1,17 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  p {
+    margin: 20px;
+    padding: 10px;
+    width: 3px;
+    border-style: double;
+    color: darkgray;
+  }
+button }
```

Comment:
Hi Fernando, for this part you will want to only add the CSS properties, so you can remove the `p {` and `button }` parts from lines 10 and 17